### ISSUE TITLE
unexpected property message in trigger

### ISSUE THREAD

**Issue by denniswham** [keesschollaart81/vscode-home-assistant#50]
With the trigger below, I get an unexpected property message in VS Code. The _for_ property seems to be the problem.

```
  trigger:
  - platform: template
    value_template: "{% if is_state_attr('proximity.home', 'dir_of_travel', 'towards') %}false{% endif %}"
    for: '00:05:00'
```

Checking the configuration doesn't return a error message. And according to the document this _for_ property is correct however: https://www.home-assistant.io/docs/automation/trigger/#template-trigger

**Comment by nierob** [comment]
hehehe I have hit the same: https://github.com/keesschollaart81/vscode-home-assistant/pull/51

**Comment by keesschollaart81** [comment]
Fixed with v1.0.1
